scripts/check_configs.ts

```diff
@@ -596,6 +675,16 @@ async function main() {
         outcomeMap[network] = false;
       }
 
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
+        const polynetworkBridges = bridgesMap.polynetwork.map((bridge) => bridge.bridgeAddresses).flat()
+        const ibcBridges = bridgesMap.ibc.map((bridge) => bridge.bridgeAddresses).flat()
```

Comment:
IBC Bridges have no bridgeAddresses
![image](https://github.com/user-attachments/assets/c27ce814-194e-4272-beff-0714a580f67e)

Maybe you can use the IBC chain id instead (i.e. `bridge.chain_id_name`)